x2b_default_config.toml

```diff
@@ -0,0 +1,16 @@
+[slurm-args]
+time = "04:00:00"
+mem = 16000
+nodes = 1
+cpus-per-task = 2
+job-name = "xnat2bids"
+output = "/gpfs/data/bnc/scratch/logs/%J.txt"
```

Comment:
This should be in the user's scratch folder. Scratch is a different partition of the file system in Oscar that is tuned for faster i/o than `/gpfs/data`. We also don't want to have users logs in our bnc folder